relion/__init__.py

```diff
@@ -61,6 +61,13 @@ def getEnviron(cls):
         cudaLib = environ.getFirst((RELION_CUDA_LIB, 'CUDA_LIB'))
         environ.addLibrary(cudaLib)
 
+        if 'RELION_MPI_LIB' in os.environ:
```

Comment:
I haven't seen this before.. are those specific vars just for relion mpi?

Reply:
I added this similarly to CUDA package specific libraries. I have found a case where Scipion/Xmipp was compiled with OpenMPI, but Relion itself was in the system compiled with MPICH, so I needed a way to specify different MPI lib.